kwok/charts/crds/karpenter.sh_nodeoverlays.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.15.0
+  name: nodeoverlays.karpenter.sh
+spec:
+  group: karpenter.sh
+  names:
+    categories:
+    - karpenter
+    kind: NodeOverlay
+    listKind: NodeOverlayList
+    plural: nodeoverlays
+    singular: nodeoverlay
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              pricePercent:
+                description: PricePercent modifies the price of the simulated node
+                  (PriceAdjustment + (Price * PricePercent / 100)).
+                type: integer
```

Comment:
If we can use a [quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/) here, it'll allow adjustments finer than 0.01
Maybe `priceMultiplier` (default value `1`)?

(if we don't, it could end up hard to change that later)

Reply:
I thought about using quantities and a multiplier, but the ergonomics were really confusing. e.g., what does it mean to have a multiplier of 100m. It's 1/10, but a milli multiplier is very unintuitive to me. Do you think we will see use cases of less than 1% increments?